conda_build/skeletons/rpm.py

```diff
@@ -52,6 +52,9 @@
     build:
       noarch: generic
 {depends_run}
+    test:
+      commands:
+        - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
```

Comment:
Ray can/should correct me if I'm wrong here, but I think CDTs are not tied strictly to that folder, and we can have them for Power, too, for example.  Maybe we need selectors here for expected behavior on each platform that uses CDTs?

Reply:
@msarahan you are correct this path is different on non-x86_64 platforms like aarch64 or ppc64le.

Reply:
That seems like a reasonable concern to raise. However there is already [hard-coding happening in the `build.sh`]( https://github.com/conda/conda-build/blob/a7ac3db661a59db7bd04d315b4a8ee739812f52c/conda_build/skeletons/rpm.py#L63-L64 ). Perhaps the right course of action is to raise an issue and address this concern separately?

Reply:
Note: This diff is outdated, but the concern raised is not currently addressed.

Reply:
Raised issue ( https://github.com/conda/conda-build/issues/3556 ) to discuss how we should address this.